### DISCUSSION TITLE
Welcome to All-Url-Uploader Discussions!

### DISCUSSION THREAD

**Discussion by kalanakt** [kalanakt/All-Url-Uploader#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by royrajan95** [comment]
Hi, I wanna know that can I deploy in vps??

**Reply by kalanakt** [reply]
Now, it's not possible. But I'll ( today or tomorrowl ) vps setup to repo then u can deploy to vps

**Reply by kalanakt** [reply]
You can use VPS, read https://github.com/kalanakt/All-Url-Uploader/tree/main/Uploader, and https://github.com/kalanakt/All-Url-Uploader READ.ME s

**Reply by royrajan95** [reply]
Thankyou very much!! u all r great...

**Reply by royrajan95** [reply]
hi,
back here to inform some bugs,

![photo_2022-08-06_22-48-26](https://user-images.githubusercontent.com/84740586/183259262-157bbebb-bfa4-41e4-b8b2-97fc6c30a0dc.jpg)
SMEDIA button is working & VIDEO button is not working...

![photo_2022-08-06_22-48-13](https://user-images.githubusercontent.com/84740586/183259306-d44563ec-72a7-46aa-871e-ec44a19f6007.jpg)
SMedia button is giving this file👆

and while selecting VIDEO button, bot wont respond, log shows 👇
![putty error](https://user-images.githubusercontent.com/84740586/183259356-ba22440c-ad4b-4281-89ba-4401dbd3414a.JPG)

Also can you please add thumbnail support, document mode support...

**Comment by arifulbashar1outlook** [comment]
could you please add thumbnail options
